README.md

```diff
@@ -21,17 +21,18 @@ Usage:
   venom run [flags]
 
 Flags:
-      --details string        Output Details Level : low, medium, high (default "medium")
-      --env                   Inject environment variables. export FOO=BAR -> you can use {{.FOO}} in your tests (default true)
-      --exclude stringSlice   --exclude filaA.yaml --exclude filaB.yaml --exclude fileC*.yaml
-      --format string         --format:yaml, json, xml, tap (default "xml")
-      --log string            Log Level : debug, info or warn (default "warn")
-      --output-dir string     Output Directory: create tests results file inside this directory
-      --parallel int          --parallel=2 : launches 2 Test Suites in parallel (default 1)
-      --resume                Output Resume: one line with Total, TotalOK, TotalKO, TotalSkipped, TotalTestSuite (default true)
-      --resumeFailures        Output Resume Failures (default true)
-      --var stringSlice       --var cds='cds -f config.json' --var cds2='cds -f config.json'
-      --var-from-file         --var-from-file filename.yaml : yaml|json, must contains map[string]string
+      --details string         Output Details Level : low, medium, high (default "low")
+      --env                    Inject environment variables. export FOO=BAR -> you can use {{.FOO}} in your tests (default true)
+      --exclude stringSlice    --exclude filaA.yaml --exclude filaB.yaml --exclude fileC*.yaml
+      --format string          --format:yaml, json, xml, tap (default "xml")
+      --log string             Log Level : debug, info or warn (default "warn")
+      --no-check-variables     Don't check variables before run
+      --output-dir string      Output Directory: create tests results file inside this directory
+      --resume                 Output Resume: one line with Total, TotalOK, TotalKO, TotalSkipped, TotalTestSuite
```

Comment:
add --parallel int          --parallel=2 : launches 2 Test Suites in parallel (default 1)